circe/.js/src/test/scala/laserdisc/interop/CirceSpecJs.scala

```diff
@@ -6,6 +6,11 @@ import munit.FunSuite
 
 final class CirceSpecJs extends FunSuite with EitherTestSyntax {
   test("handling an invalid json fails to decode") {
-    Read[Bulk, Bar].read(Bulk("{")) onLeft (e => assertEquals(e.message, "ParsingFailure: SyntaxError: Unexpected end of JSON input"))
+    Read[Bulk, Bar].read(Bulk("{")) onLeft { e =>
+      assertEquals(
+        e.message,
+        "ParsingFailure: SyntaxError: Expected property name or '}' in JSON at position 1"
```

Comment:
this is an interesting one. Apparently in the CI the scala.js parsing fails with
```
ParsingFailure: SyntaxError: Unexpected end of JSON input
```
and locally on my env it fails with
```
ParsingFailure: SyntaxError: Expected property name or '}' in JSON at position 1
```
I couldn't track it to a specific version of runtime of some sort yet

Reply:
so the CI runs node v16.18.1 (from [here](https://github.com/actions/runner-images/blob/ubuntu22/20221127.1/images/linux/Ubuntu2204-Readme.md)) that gives that error while I was running v19.2.0 and that fails with the other. I will change the test tomorrow to avoid these shenanigans.